how-to/workspace-platform-starter/client/src/modules/integrations/apps/integration.ts

```diff
@@ -7,12 +7,7 @@ import type {
 	HomeSearchResponse,
 	HomeSearchResult
 } from "@openfin/workspace";
-import type {
-	NotificationCreatedEvent,
-	NotificationOptions,
-	NotificationsCountChanged
-} from "@openfin/workspace/notifications";
-import type { FavoriteChangedLifecyclePayload, NotificationClient } from "workspace-platform-starter/shapes";
+import type { FavoriteChangedLifecyclePayload } from "workspace-platform-starter/shapes";
```

Comment:
Be more specific about the import i.e. add `/lifecycle-shapes`

Reply:
That's your own code :)